research/minigo/minigo.py

```diff
@@ -196,31 +174,33 @@ def gather(selfplay_dir, training_chunk_dir, params):
     f.write('\n'.join(sorted(already_processed)))
 
 
-def train(trained_models_dir, estimator_model_dir, training_chunk_dir, params):
+def train(trained_models_dir, estimator_model_dir, training_chunk_dir,
+          generation_num, params):
   """Train the latest model from gathered data.
 
   Args:
     trained_models_dir: Where to export the completed generation.
     estimator_model_dir: tf.estimator model directory.
     training_chunk_dir: Directory where gathered training chunks are.
+    generation_num: Which generation you are training.
```

Comment:
I feel "generation" is enough for indicating it is a number and integer.

Reply:
Sure. Will update it.